diffsync/store/redis.py

```diff
@@ -18,10 +18,13 @@
 class RedisStore(BaseStore):
     """RedisStore class."""
 
-    def __init__(self, *args, store_id=None, host="localhost", port=6379, url=None, db=0, **kwargs):
+    def __init__(self, *args, store_id=None, host=None, port=6379, url=None, db=0, **kwargs):
         """Init method for RedisStore."""
         super().__init__(*args, **kwargs)
 
+        if url and host and port:
```

Comment:
Perhaps?

```suggestion
        if url and (host or port):
```

Reply:
if we do like this, then I have to remove the default port from the kwargs, that I thought it would be convenient. But, it could be also confusing... not sure

Reply:
OK, perhaps just `if url and host` then?